docs/mint.json

```diff
@@ -70,6 +70,10 @@
     {
       "name": "All Tools",
       "url": "https://composio.dev/tools"
+    },
+    {
+      "name": "Chat with Code",
+      "url": "https://chatgpt.com/g/g-67697db23c808191a1787ea4b86ac1ce-composio"
```

Comment:
Please ensure that the URL provided for 'Chat with Code' is correct and accessible. It's important to verify that external links are functional and lead to the expected content to avoid user confusion or broken links in the documentation.